toolchain/sem_ir/expr_info.h

```diff
@@ -42,6 +42,11 @@ enum class ExprCategory : int8_t {
 // Returns the expression category for an instruction.
 auto GetExprCategory(const File& file, InstId inst_id) -> ExprCategory;
 
+// Returns whether the given expression category is for a reference expression.
+inline auto IsReferenceCategory(ExprCategory cat) -> bool {
```

Comment:
Shouldn't `Ref` be abbreviated in the name here?
```suggestion
inline auto IsRefCategory(ExprCategory cat) -> bool {
```